websites/I/IFTTT/presence.ts

```diff
@@ -1,277 +1,361 @@
-                break;
-            }
-          }
-          break;
-        //My services
-        case "date_and_time":
-        case "email":
-        case "email_digest":
-        case "ifttt":
-        case "feed":
-        case "space":
-        case "weather":
-        case "maker_webhooks":
-          presenceData.details = "My Services";
-          presenceData.state = document.querySelector("h1").textContent;
-          break;
-        case "my_services":
-          presenceData.details = "My Services";
-          break;
-        //Connect services
-        case "connect":
-          presenceData.details = "My Services";
-          presenceData.state = `Connect ${
-            (document.querySelector("h1").firstElementChild as HTMLTitleElement)
-              .title
-          } to ${
-            (document.querySelector("h1").lastElementChild as HTMLTitleElement)
-              .title
-          }`;
-          break;
-        //Explore
-        case "discover":
-          presenceData.details = "Explore";
-          break;
-        //Search
-        case "search":
-          presenceData.smallImageKey = "search";
-          presenceData.smallImageText = (await strings).search;
+/**
+ * Handle service page
+ */
+async function handleSerivce(): Promise<void> {
+  presence.setActivity({
+    details: document.querySelector<HTMLHeadingElement>("h1").textContent,
+    state: document.querySelector<HTMLImageElement>(".large-service-logo")
+      .title,
+    largeImageKey: "logo_big",
+    smallImageText: (await strings).browsing,
+    smallImageKey: "reading"
+  });
+}
 
-          presenceData.details = `Searching for "${
-            (document.getElementById("search") as HTMLInputElement).value
-          }"`;
-          presenceData.state = `Tab: ${
-            document.getElementsByClassName("active")[1].textContent
-          }`;
-          break;
-        //Trust & Privacy
-        case "terms":
-          presenceData.details = "Privacy Policy & Terms of Use";
-          break;
-        //Careers
-        case "careers":
-          presenceData.details = "Careers";
-          break;
-        //Unknown
-        default:
-          if (document.getElementsByClassName("brand-section").length > 0) {
-            presenceData.details = "Services";
-            presenceData.state = document.querySelector("h1").textContent;
-          } else {
-            presence.setActivity();
-            return;
-          }
-      }
-      break;
-  }
+/**
+ * Handle search page
+ * @param value Search value
+ */
+async function handleSearch(value: string): Promise<void> {
+  presence.setActivity({
+    details: "Searching for Applets & Services",
+    state: value,
+    largeImageKey: "logo_big",
+    smallImageText: (await strings).search,
+    smallImageKey: "search"
+  });
+}
 
-  presence.setActivity(presenceData);
-});
+/**
+ * Handle blog pages
+ * @param entry Blog entry name
+ */
+async function handleBlog(entry?: string): Promise<void> {
```

Comment:
Does not await anything so it does not need to be async nor does it need to have a type
```suggestion
function handleBlog(entry?: string) {
```